dockerfiles/jupyter/mlcestart.sh

```diff
@@ -0,0 +1,16 @@
+file_path="${HOME}/.intdata"
+
+if [ -f "$file_path" ]; then
+    echo "base data exists."
+    # Perform actions when the file exists
+else
+    echo "base data does not exist."
+    echo "1" > $file_path
+    echo "base data exists."
+    cd / ; tar -xvf /tmp/basehome.tar
+    # Perform actions when the file does not exist
+fi
+
+cd ${HOME}
+
+start-notebook.sh  --ip=0.0.0.0 --port=8888 --NotebookApp.token="" --NotebookApp.password="" --NotebookApp.default_url=/lab
```

Comment:
what about the mlrun db? you need to run them in parallel like in the dockerfile

Reply:
not sure i understand the comment